### PR TITLE
build the kustomize output to Infra-deployment deploy file

### PR DESCRIPTION
**Pull request by prietyc123**
The pipeline-service application is now updated via deploy file after pr: https://github.com/redhat-appstudio/infra-deployments/pull/2383

So adding the kustomize output in the infra-deployment-update-script so that the bot also update the deploy file for staging folder of pipeline-service

### CODE REVIEW

**Comment by prietyc123** [comment]
/hold

Team, PR is blocked on [image reference](https://github.com/redhat-appstudio/build-definitions/blob/main/task/update-infra-deployments/0.1/update-infra-deployments.yaml#L28C24-L28C32). Started slack [thread](https://redhat-internal.slack.com/archives/C04GVLR0155/p1700750873798359) on this

**Comment by prietyc123** [comment]
> /hold
> 
> Team, PR is blocked on [image reference](https://github.com/redhat-appstudio/build-definitions/blob/main/task/update-infra-deployments/0.1/update-infra-deployments.yaml#L28C24-L28C32). Started slack [thread](https://redhat-internal.slack.com/archives/C04GVLR0155/p1700750873798359) on this

PR https://github.com/redhat-appstudio/build-definitions/pull/685

File: .tekton/open-infra-deployment-pr.yaml
Comment (Roming22) [inline .tekton/open-infra-deployment-pr.yaml:44]: What is the purpose of that new param? I don't see it being used.

Comment (prietyc123) [reply .tekton/open-infra-deployment-pr.yaml:44]: > What is the purpose of that new param? I don't see it being used.

To get the build output in [deploy.yaml](https://github.com/redhat-appstudio/infra-deployments/pull/2383/files#diff-5a2a995c501aca7ed40fb6724a47d82f3f4a4db2cc8b4c3677cfffecfa39b507) file, we need `kustomize` cli but in the existing task [script image](https://github.com/redhat-appstudio/build-definitions/blob/main/task/update-infra-deployments/0.1/update-infra-deployments.yaml#L28), it does not have kustomize binary. Basically we need an image which has `kustomize` as well, also I don't want everyone to get affected with this image change.

> And by the way, somebody needs to minimally pull down quay.io/mkovarik/ose-cli-git:4.11 and store it in a quay repo other that Michal Kovarik's, who no longer is at Red Hat. What if he decides to delete his quay account?

+1 to this

**Review by Roming22 — APPROVED** [review]
LGTM

@prietyc123 Just check with Xin before merging. He's troubleshooting the CI and a merge event might throw a wrench in his debugging process.

**Comment by Roming22** [comment]
This PR does not impact the CI. The failing test is failing because of an unrelated issue.
PR can be merged as is.

**Pull request merged**
